### PR TITLE
challenge2

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: biggestNum/testbiggest.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const test = (num1,num2,num3,result)=>{
+  if(biggest(num1,num2,num3)==result){
+    console.log('correct :' + result);
+  }else{
+    console.log('wrong :' + result);
+  }
+}
+
+test(3,6,1,6);
```
Comment (seemcat) [inline biggestNum/testbiggest.js:11]: Write at least 2 more tests.

**Review by seemcat — CHANGES_REQUESTED** [review]

**Review by allopez7 — APPROVED** [review]

**Review by seemcat — APPROVED** [review]

File: biggestNum/biggestnum.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const biggest=(num1,num2,num3)=>{
+  if(num1>num2 && num1>num3){
+    return num1;
+  }
+  if(num2>num1 && num2>num3){
+    return num2;
+  }
+  if(num3>num1 && num3>num2){
```
Comment (chocobrownie) [inline biggestNum/biggestnum.js:8]: just use return num3 straight

File: biggestNum/biggestnum.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const biggest=(num1,num2,num3)=>{
+  if(num1>num2 && num1>num3){
+    return num1;
+  }
+  if(num2>num1 && num2>num3){
```
Comment (chocobrownie) [inline biggestNum/biggestnum.js:5]: else if (num2 > num3)

**Comment by songz** [comment]
can you delete your folder and rename your file to the issue number?

**Pull request closed without merging**